cmd/get_logs.go

```diff
@@ -347,6 +380,28 @@ func (glc *getLogsCmd) downloadLogs(node v1.Node, client *ssh.Client) (string, e
 	return "", nil
 }
 
+func (glc *getLogsCmd) validateLogScript() error {
+	if glc.linuxScriptPath == "" && !glc.cs.Properties.MasterProfile.IsVHDDistro() {
+		if glc.controlPlaneOnly {
+			return errors.Errorf("No log collection script found for control plane nodes")
+		} else {
+			log.Warnf("Skipping master nodes nodes as no log collection script found")
+			glc.masterNodes = nil
```

Comment:
is this working? I think you should set it to an empty slice

Reply:
Yes this is working and this is also how an empty slice of specific type looks like when initially created.